django/db/models/fields/__init__.py

```diff
@@ -843,18 +866,12 @@ def get_flatchoices(self, include_blank=True,
         first_choice = blank_choice if include_blank else []
         return first_choice + list(self.flatchoices)
 
-    def _get_val_from_obj(self, obj):
```

Comment:
If possible, it would be great to try to submit cleanups like this as separate pull requests that can be merged ahead of the main composite field work. Otherwise, I fear we will end up with a monolithic pull request that will be very difficult to review.